client/asset/dcr/config.go

```diff
@@ -33,12 +33,12 @@ var (
 // Config holds the parameters needed to initialize an RPC connection to a dcr
 // wallet. Default values are used for RPCListen and/or RPCCert if not set.
 type Config struct {
-	RPCUser         string `ini:"username"`
-	RPCPass         string `ini:"password"`
-	RPCListen       string `ini:"rpclisten"`
-	RPCCert         string `ini:"rpccert"`
-	UseSplitTx      bool   `ini:"txsplit"`
-	FallbackFeeRate uint64 `ini:"fallbackfee"`
+	RPCUser         string  `ini:"username"`
+	RPCPass         string  `ini:"password"`
+	RPCListen       string  `ini:"rpclisten"`
+	RPCCert         string  `ini:"rpccert"`
+	UseSplitTx      bool    `ini:"txsplit"`
+	FallbackFeeRate float64 `ini:"fallbackfee"`
```

Comment:
Do we want this to match dcrwallet's `txfee` setting?

`--txfee=                                 Transaction fee per kilobyte (default: 0.0001 DCR)`

dcrwallet doesn't have a `fallbackfee` setting and I'm honestly not sure what happens if fee estimation fails with dcrwallet or even which txn rpcs try to estimate a smart fee vs just using `txfee`.

Reply:
`estimatesmartfee` is actually **dcrd**, and it looks like it can only error [in `estimateMedianFee` here](https://github.com/decred/dcrd/blob/125fc41c76b19b35994d0885571a93c25dd9489e/internal/fees/estimator.go#L647), and probably can't happen on mainnet or testnet3.

Reply:
I was just referring to the way bitcoin _wallets_ use `fallbackfee` for authoring transactions with the wallet when fee estimation fails, not just with the `estimatesmartfee` directly.  With decred, it's dcrwallet making transactions using `txfee` presumably (although smart fee estimation would be preferred when dcr fees get higher).  So I was just wondering if it makes sense to use `txfee` from a dcrwallet config to get fallbackfee since "fallbackfee" is only a bitcoin thing.

Reply:
We're not using it as a default transaction fee in the same way as dcrwallet, but as a value to use when `estimatesmartfee` fails, which is how it's used for bitcoin, so I thought retaining the name made sense.

Reply:
>  as a value to use when `estimatesmartfee` fails

Which also means that it'll probably never be used on mainnet anyways, if that means anything.

Reply:
Yeah, I don't have a problem keeping the name.  But estimatesmartfee falis on mainnet until the node has a chance to watch mempool and a few blocks go by.